### PR TITLE
Fix for PR#14

### PR DESCRIPTION
**Pull request by giorgiopizz**
This PR implements fix for:
* mkShapesRDF.py
  * with default runner
  * global variables not being picked up
  * Nice formatting of batch submitted samples that are running or are done (mkShapesRDF -o 1)
* histo_utils.py: fix to folding
* Fix to Full2018v9 example folder

### CODE REVIEW

**Comment by mlizzo** [comment]
Hi @giorgiopizz , fixes look good to me, PR can be merged if no one has further comments

**Comment by NTrevisani** [comment]
Thank you @giorgiopizz and @mlizzo. If the PR looks good for you, please proceed and merge it.

File: examples/Full2018_v9/nuisances.py [hunk 1]
```diff
@@ -1,259 +1,256 @@
...
+# limitFiles = -1
 
 print(treeBaseDir)
-def makeMCDirectory(var=''):
-    _treeBaseDir = treeBaseDir + ''
+
+
+def makeMCDirectory(var=""):
+    _treeBaseDir = treeBaseDir + ""
     if useXROOTD:
```
Comment (GiacomoBoldrini) [inline examples/Full2018_v9/nuisances.py:14]: Not defined?

**Comment by NTrevisani** [comment]
Hi! Anybody against merging this PR?

**Comment by GiacomoBoldrini** [comment]
Two very tiny changes in order to make things work out of the box.

@giorgiopizz can you add to this PR the change in the usage of `useXROOTD` in the examples + more importantly the handling of missing nuisance templates in PlotFactory? 

To be clear:
[This line ](https://github.com/latinos/mkShapesRDF/blob/e84da5b3cf831e5f7e038e95d197478eb9ca5070/mkShapesRDF/shapeAnalysis/latinos/PlotFactory.py#L678) should be changed to `if histoVar != None:` due to the python handling of ROOT TObjects (see [here](https://root-forum.cern.ch/t/findobject-returns-a-null-pointer-tobject/8408) for an example)

**Comment by giorgiopizz** [comment]
Hi, new fixes are implemented, namely:
* useXROOTD and the usage of useXROOTD [here](https://github.com/latinos/mkShapesRDF/pull/15/files#diff-1f280be9218100a61dd483ec92ff6f3feb9207f832ce86f06bf36b1fd2a34178R16-R34) and fixes to the example `Full2018_v9`
* pdfs/qcd_scales histos are not removed in postProcessNuisance
* The raise in `rnp`  is fixed
@GiacomoBoldrini @NTrevisani @mlizzo could you give it a look please?

**Comment by GiacomoBoldrini** [comment]
2D, 3D unrolling is still not working properly as bin counts and bin errors are not unrolled correctly. We will need to open another PR to fix it

**Comment by mlizzo** [comment]
Thanks @GiacomoBoldrini , we will try to address the issue in the next PR

**Comment by giorgiopizz** [comment]
Working on it

**Pull request merged**